exercises/protein-translation/src/test/scala/ProteinTranslationTest.scala

```diff
@@ -0,0 +1,65 @@
+import org.scalatest.{FunSuite, Matchers}
```

Comment:
Well, there is an [open issue](https://github.com/exercism/x-common/issues/577), but as said, getting an agreement on the data could take a while. So perhaps just continue as is for the moment?